packages/admin-ui/ui/src/domain/product-categories/modals/add-product-category.tsx

```diff
@@ -90,9 +91,20 @@ function CreateProductCategory(props: CreateProductCategoryProps) {
 
       <FocusModal.Main className="no-scrollbar flex w-full justify-center">
         <div className="small:w-4/5 medium:w-7/12 large:w-6/12 my-16 max-w-[700px]">
-          <h1 className="inter-xlarge-semibold text-grey-90 pb-8">
+          <h1 className="inter-xlarge-semibold text-grey-90 pb-10">
```

Comment:
suggestion: spacing of this component between the header and the next section is a bit odd (at least to me) and differs a bit from the design